scripts/min.cmd

```diff
@@ -37,7 +37,7 @@ match 1.tmp 'MM>' n
 match 1.tmp '  ' y
 break 1.tmp ''
 unduplicate 1.tmp
-add 1.tmp 'prove ' '$min xxx/allow */no_new ax-*$save new_proof/compressed$_exit_pa$'
+add 1.tmp 'prove ' '$min xxx/allow */no_new ax-*/may$min *$save new_proof/compressed$_exit_pa$'
```

Comment:
What does this enhancement mean?

Reply:
Sorry, this was meant to be temporary and not added to the PR. I'm going to remove it.

Reply:
Done.